src/ontology/fbbt-edit.obo

```diff
@@ -117740,7 +117740,7 @@ is_a: FBbt:00007357 ! basiconic sensillum LB-II4
 [Term]
 id: FBbt:00067013
 name: adult olfactory receptor neuron Or10a
-def: "Bilateral olfactory receptor neuron (ORN) that expresses Or10a (FBgn0030298) and the gustatory receptor Gr10a (FBgn0045502) (Fishilevich and Vosshall, 2005). Its axon branches, with one branch innervating the ipsilateral antennal lobe glomerulus DL1 and the other fasciculating with the antennal commissure and innervating the contralateral antennal lobe glomerulus DL1. Its dendrite transduces signals from antennal basiconic sensillum ab1." [DoOR:Or10a, FlyBase:FBrf0179238, FlyBase:FBrf0187305, FlyBase:FBrf0187306]
+def: "Bilateral olfactory receptor neuron (ORN) that expresses Or10a (FBgn0030298) and the gustatory receptor Gr10a (FBgn0045502) (Fishilevich and Vosshall, 2005). Its axon branches, with one branch innervating the ipsilateral antennal lobe glomerulus DL1 and the other fasciculating with the antennal commissure and innervating the contralateral antennal lobe glomerulus DL1. Its dendrite transduces signals from antennal basiconic sensillum ab1. The numbers of those neurons have been estimated to 37.5±6.5 in females and 24±4 in males (Grabe et al., 2016)." [DoOR:Or10a, FlyBase:FBrf0179238, FlyBase:FBrf0187305, FlyBase:FBrf0187306, FlyBase:FBrf0233453]
```

Comment:
It is usually best to avoid special characters like this. I think in this case it would be good enough to say 'approximately 37.5' (or round to 38) and 'approximately 24'. Please change all uses of '±' - probably easiest to do this in a text editor.